### ISSUE TITLE
Missing dependency declarations in some servers

### ISSUE THREAD

**Issue by JonKrone** [modelcontextprotocol/servers#32]
Hey y'all, a few of the MCP server packages are missing required dependencies in their `package.json` files, causing runtime errors when ran via `npx`.

### What I'm seeing
```sh
> npx @modelcontextprotocol/server-filesystem@latest
node:internal/modules/esm/resolve:839
  throw new ERR_MODULE_NOT_FOUND(packageName, fileURLToPath(base), null);
        ^

Error [ERR_MODULE_NOT_FOUND]: Cannot find package 'zod-to-json-schema' imported
from /Users/.../@modelcontextprotocol/server-filesystem/dist/index.js
```

### Affected Packages
- `@modelcontextprotocol/server-filesystem`
- `@modelcontextprotocol/server-github`
- `@modelcontextprotocol/server-google-maps`

I'm not seeing any errors having added the `sqlite`, `postgres`, and `memory` servers. There may be missing deps for other MCP packages.

### Missing deps:
Different packages are missing different dependencies but I've seen at least these two missing:

- `zod-to-json-schema`
- `node-fetch`

### Suggested Fix
I believe the fix is just to declare these dependencies in the package.json.

### Reproduction
1. Run the package via npx: `npx @modelcontextprotocol/server-filesystem`
2. Observe dependency errors

### Environment
- Node.js version: 22.9.0
- npm version: 10.8.3
- OS: macOS

![image](https://github.com/user-attachments/assets/2b36f64a-0e36-4c0a-b3dc-6e09ea8e2491)

**Comment by jspahrsummers** [comment]
Thanks for flagging! A downside of using npm workspaces is that it's easy to miss stuff like this. 😰 

Will try to figure out a better CI story too.

**Comment by jspahrsummers** [comment]
Should be fixed in 0.5.1 of these servers, just published. Sorry about the trouble!
